src/main/java/domain/AttendanceAnalyze.java

```diff
@@ -0,0 +1,51 @@
+package domain;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class AttendanceAnalyze implements Comparable<AttendanceAnalyze> {
+    private final int lateCount;
+    private final int absenceCount;
+    private final int attendanceCount;
+
+    public AttendanceAnalyze(final List<AttendanceHistory> histories) {
+        Map<AttendanceResult, Integer> statics = histories.stream().collect(Collectors.groupingBy(
+                AttendanceHistory::getAttendanceResult,
+                () -> new EnumMap<>(AttendanceResult.class),
+                Collectors.summingInt((e) -> 1)
+        ));
```

Comment:
stream의 체이닝 메서드들은 한 줄에 하나의 메서드만 쓰도록해보면 어떨까요? 😃
가독성적인 측면에서도 더 좋고 체이닝 메서드를 3개이상 호출할 경우 각 메서드 체이닝을 호출했을 때의 결과 타입을 
IntelliJ가 보여주기 때문에 조금 더 코드를 파악하는데에도 도움이된다고 생각해요. 😃 

```suggestion
        Map<AttendanceResult, Integer> statics = histories.stream()
            .collect(Collectors.groupingBy(
                AttendanceHistory::getAttendanceResult,
                () -> new EnumMap<>(AttendanceResult.class),
                Collectors.summingInt((e) -> 1)
        ));
```

Reply:
확실히 로키의 코드가 스트림의 가독성 장점을 돋보이게 하는거 같습니다! 생각해보니 요구사항에서 한 줄에 . 하나만 찍는다는 점도 위배하고 있네요! 수정해보도록 하겠습니다!